content/articles/spring-webflux/index.md

```diff
@@ -0,0 +1,315 @@
+Reactive programming is a programming paradigm that supports an asynchronous, event-driven, and non-blocking approach to data processing. The reactive programming paradigm organizes events and data as streams.
+
+In the reactive programming paradigm, when a request is made, other tasks are executed while waiting for the results. When the data is available, a notification is sent with the data through the callback function. The reactive programming paradigm is suitable for data-driven applications such as chat applications.
+
+In this tutorial, we are going to create a student management system using Spring Webflux and MongoDB.
+
+
+### Prerequisites
+- [JDK](https://www.oracle.com/java/technologies/javase-downloads.html) installed on your computer.
+- Your favorite IDE or editor installed.
+- Knowledge of [Java](https://www.tutorialspoint.com/java/index.htm) and [Spring Boot](https://spring.io/projects/spring-boot).
+- Knowledge of [Mongodb](https://www.mongodb.com/).
+
+### Streams API
+Software developers at Netflix, Twitter, Pivotal, and Redhat created the streams API. Streams API defines four interfaces below.
+
+#### Publisher
+The publisher interface emits events to subscribers based on the request sent by the subscribers. Thus, a single publisher can serve several subscribers.
+
+```java
+public interface Publisher<T> 
+{
+    public void subscribe(Subscriber<? super T> s);
+}
+```
+
+#### Subscriber
+The `Subscriber` interface listens and receives events from the `Publisher` interface. The `Subscriber` interface has four methods to handle the response from the `Publisher` interface.
+
+```java
+public interface Subscriber<T> 
+{
+    public void onSubscribe(Subscription s);
+    public void onNext(T t);
+    public void onError(Throwable t);
+    public void onComplete();
+}
+```
+
+#### Subscription
+The `Subscription` interface defines a one-to-one relationship between the `Publisher` and `Subscriber` interfaces. It can be used to request data and also cancel the request.
+
+```java
+public interface Subscription<T> 
+{
+    public void request(long n);
+    public void cancel();
+}
+```
+
+#### Processor
+The `Processor` interface represents the processing stage containing both the `Publisher` and the `Subscriber`.
+
+```java
+public interface Processor<T, R> extends Subscriber<T>, Publisher<R> 
+{
+
+}
+```
+
+> There are two popular implementations of the reactive streams, [RxJava](https://github.com/ReactiveX/RxJava) and [Project reactor](https://projectreactor.io/).
+
+### Spring Webflux
+Spring Webflux is similar to Spring MVC, but it supports reactive and non-blocking streams.
+
+Webflux has two publishers:
+
+**Mono**
```

Comment:
You forgot to change the Mono and Flux subheaders to use four \#s instead of asterisks. So for instance, you would change this on line 68:

> \*\*Mono\*\*

To this:

> \#\#\#\# Mono

And likewise, change this on line 75:

>\*\*Flux\*\* 

To this:

>\#\#\#\# Flux